test/byContract/IexecPoco/03_reveal.test.ts

```diff
@@ -97,195 +97,209 @@ describe('IexecPoco2#reveal', () => {
+        // but contribution outside consensus
+        await expect(
+            iexecPoco.connect(loosingWorker).reveal(taskId, resultDigest),
+        ).to.be.revertedWithoutReason(); // require#4
+    });
 
-        it('Should not reveal when no able to prove result ownership', async () => {
-            await iexecPocoAsWorker
-                .contribute(
-                    taskId,
-                    resultHash,
-                    ethers.utils.hexZeroPad('0xbad5ea1', 32), // bad seal
-                    emptyEnclaveAddress,
-                    emptyEnclaveSignature,
-                    schedulerSignature,
-                )
-                .then((tx) => tx.wait());
-            const task = await iexecPoco.viewTask(taskId);
-            expect(task.status).equal(TaskStatusEnum.REVEALING);
-            const contribution = await iexecPoco.viewContribution(taskId, worker.address);
-            expect(contribution.status).equal(ContributionStatusEnum.CONTRIBUTED);
-            expect(contribution.resultHash).equal(task.consensusValue);
-            expect(contribution.resultHash).equal(buildResultHash(taskId, resultDigest));
-            expect(contribution.resultSeal).not.equal(
-                buildResultHashAndResultSeal(taskId, resultDigest, worker),
-            );
-            // revealing task, before deadline, contribution status is contributed
-            // contribution is part of the consensus, result proof is valid
-            // but unable to prove result ownership
-            await expect(
-                iexecPocoAsWorker.reveal(taskId, resultDigest),
-            ).to.be.revertedWithoutReason(); // require#6
-        });
+    it('Should not reveal when unable to prove result value', async () => {
+        await iexecPocoAsWorker
+            .contribute(
+                taskId,
+                resultHash,
+                resultSeal,
+                emptyEnclaveAddress,
+                emptyEnclaveSignature,
+                schedulerSignature,
+            )
+            .then((tx) => tx.wait());
+        const task = await iexecPoco.viewTask(taskId);
+        expect(task.status).equal(TaskStatusEnum.REVEALING);
+        const contribution = await iexecPoco.viewContribution(taskId, worker.address);
+        expect(contribution.status).equal(ContributionStatusEnum.CONTRIBUTED);
+        expect(contribution.resultHash).equal(task.consensusValue);
+        expect(contribution.resultHash).not.equal(buildResultHash(taskId, badResultDigest));
+        // revealing task, before deadline, contribution status is contributed
+        // contribution is part of the consensus
+        // but unable to prove result value
+        await expect(
+            iexecPocoAsWorker.reveal(taskId, badResultDigest),
+        ).to.be.revertedWithoutReason(); // require#5
+    });
+
+    it('Should not reveal when no able to prove result ownership', async () => {
```

Comment:
```suggestion
    it('Should not reveal when unable to prove result ownership', async () => {
```

Reply:
I missed it thanks! [Update test wording](https://github.com/iExecBlockchainComputing/PoCo/pull/114/commits/9fc75d4fecffb9ca1ab85ea19742c3b220e3c410)